plugins/linux-pipewire/portal.c

```diff
@@ -121,3 +130,78 @@ void portal_create_session_path(char **out_path, char **out_token)
 		bfree(sender_name);
 	}
 }
+
+static void portal_signal_call_free(struct portal_signal_call *call)
+{
+
+	if (call->signal_id)
+		g_dbus_connection_signal_unsubscribe(
+			portal_get_dbus_connection(), call->signal_id);
+
+	if (call->cancelled_id > 0)
+		g_signal_handler_disconnect(call->cancellable,
+					    call->cancelled_id);
+
+	g_clear_pointer(&call->request_path, bfree);
+	bfree(call);
+}
+
+static void on_cancelled_cb(GCancellable *cancellable, void *data)
+{
+	UNUSED_PARAMETER(cancellable);
+
+	struct portal_signal_call *call = data;
+
+	blog(LOG_INFO, "[pipewire] Request cancelled");
```

Comment:
Is the logging prefix in this module "[pipewire]" or "[pipewire]: " or something else?

Reply:
It's actually "portals"